lib/lhm/invoker.rb

```diff
@@ -52,6 +54,11 @@ def run(options = {})
       end
     end
 
+    def triggers_still_exist?(entangler)
+      triggers = connection.execute("SHOW TRIGGERS LIKE '#{migrator.origin.name}'").map {|t| t.first }
```

Comment:
Aren't you missing a '%' at the start of the string?

```
SHOW TRIGGERS LIKE '%#{migrator.origin.name}'
```

I would be as strict as possible in this case (sometimes we use PTOSC as well, which also creates triggers):
```
SHOW TRIGGERS LIKE 'lhmt_%#{migrator.origin.name}'
```